selectel/resource_selectel_mks_nodegroup_v1.go

```diff
@@ -159,6 +159,11 @@ func resourceMKSNodegroupV1() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validation.StringLenBetween(0, 65535),
 			},
+			"install_nvidia_device_plugin": {
+				Type:     schema.TypeBool,
+				Required: true,
+				ForceNew: true,
```

Comment:
Do we really need to create new nodegroup if this option changed ?

Reply:
Yes, because we have to change our node image (with GPU drivers / without GPU drivers). NDP installation depends on node image in create node group flow